skycultures/modern_journey_to_the_west/description.md

```diff
@@ -0,0 +1,531 @@
+# Modern Journey to the West
+
+## Introduction
+
+*Journey to the West* is a classic of Chinese God and devil novels. It is written in the 16th-century Ming dynasty. It mainly chronicles the epic pilgrimage of the monk Xuanzang (Tang Monk) as he travels westward to obtain sacred Buddhist scriptures. He is accompanied by his three formidable disciples: the rebellious and powerful Monkey King, Sun Wukong; the humorous yet often gluttonous Zhu Bajie; and the steadfast Sha Wujing. Throughout their journey, they encounter a captivating array of celestial beings, demons, and deities, battling hardships and overcoming eighty-one tribulations to reach their ultimate goal.
+
+This unique sky culture reimagines the classical Greco-Roman constellations, replacing their figures with deities, monsters, and magical artifacts from *Journey to the West*, creating an entirely new constellation world imbued with the rich hues of Chinese mythology.(The Journey to the West sky culture includes 84 of the 88 IAU standard constellations. The four constellations Circinus, Pictor, Pyxis and Telescopium have no corresponding figures or elements in the story.)
+
```

Comment:
After reading some background about the story on Wikipedia: It would be interesting to know why or how the (entirely non-astronomical) story got transferred into constellations by Xu Gang. Is there a story line which can be told, e.g. by arranging the constellations into a sequence that tells the main story? The original story was published in a time when Jesuits had come to China and describes a journey from yet ~1000 years earlier. When the Jesuits brought the Ptolemaic constellations, were they adopted early-on in some folk tradition reflecting the "Journey"'s popularity, or is the adoption Xu Gang's idea? The inclusion/adaptation of semi-modern constellations (still most of La Caille's 18th ct creations) indicates it is really a new idea.